src/Toolkit/Toolkit.iOS/UI/Controls/Compass/Compass.iOS.cs

```diff
@@ -74,7 +74,7 @@ private void Initialize()
 
         private void OnTapped()
         {
-            ResetRotation();
+            CompassClicked?.Invoke(this, EventArgs.Empty);
```

Comment:
This should still call ResetRotation. Perhaps better to rename ResetRotation to a common internal "OnCompassClicked" method, that resets (if enabled) and raises the public event, rather than subscribing to an event this control already owns (that also avoids raising events when no one externally is listening)